examples/disaggregated/README.md

```diff
@@ -19,6 +19,8 @@ cache_transceiver_config:
   # KV cache transfer timeout in milliseconds
   # For requests, if they do not send/receive the KV cache in time they are cancelled and cleaned up
   kv_transfer_timeout_ms: <int>
+  # Timeout in milliseconds to wait for the sender future to be ready when scheduled batch size is 0. This allows the request to be eventually cancelled by the user or because of kv_transfer_timeout_ms
+  kv_transfer_sender_future_timeout_ms: <int>
```

Comment:
I think this parameter is hard to understand for the user. Would it make sense to change the logic in executor loop so that we don't block if we can't find any KVCache blocks? For example, can we do polling (e.g. check KVCache status every X milliseconds)?